pyproject.toml

```diff
@@ -24,52 +27,31 @@ classifiers = [
     "Topic :: Text Processing :: Markup :: HTML",
     "Topic :: Text Processing :: Markup :: reStructuredText",
 ]
+requires-python = ">=3.8.1,<4.0"
+dependencies = [
+    "blinker>=1.4",
+    "docutils>=0.16",
+    "feedgenerator>=1.9",
+    "jinja2>=2.7",
+    "pygments>=2.6",
+    "python-dateutil>=2.8",
+    "rich>=10.1",
+    "unidecode>=1.1",
+    "backports-zoneinfo<1.0.0,>=0.2.1;python_version<'3.9'",
+    "watchfiles",
```

Comment:
We should probably specify a minimum version for `watchfiles`, as we do for all the other dependencies:

    "watchfiles>=0.21.0",